aiogram/dispatcher/dispatcher.py

```diff
@@ -126,11 +126,13 @@ async def process_update(
         :param kwargs: contextual data for middlewares, filters and handlers
         :return: status
         """
+        handled = False
         try:
-            async for result in self.feed_update(bot, update, **kwargs):
-                if call_answer and isinstance(result, TelegramMethod):
-                    await self._silent_call_request(bot=bot, result=result)
-                return True
+            response = await self.feed_update(bot, update, **kwargs)
+            handled = handled is not NOT_HANDLED
```

Comment:
very confusing line. can't we:
```suggestion
            handled = True
```
?

Reply:
What about `SKIPPED`/`IGNORED`/etc.?

`handler is not SKIPPED`